src/main/java/com/znsio/rpi/properties/Config.java

```diff
@@ -0,0 +1,43 @@
+package com.znsio.rpi.properties;
+
+import org.apache.commons.lang3.exception.ExceptionUtils;
+import org.apache.log4j.Logger;
+
+import javax.validation.constraints.NotNull;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Properties;
+
+public class Config {
+    public static final String TARGET_ENVIRONMENT = "TARGET_ENVIRONMENT";
+    public static final String PLATFORM = "PLATFORM";
+    public static final String APP_NAME = "APP_NAME";
+    public static final String IS_VISUAL = "IS_VISUAL";
+    public static final String IS_MOBILAB = "IS_MOBILAB";
+    public static final String BROWSER = "BROWSER";
```

Comment:
What if the test is for android/iOS?

Reply:
We're setting the property accordinlgy in ReportPortalConfigurationManager based on whether it is Web or App tests? We have APP_PACKAGE_NAME for android/iOS